README.md

```diff
@@ -15,23 +15,23 @@ Click [here](https://github.com/line/clay/blob/master/screenshot/example.gif) to
 
 Use Gradle to build the library. Download it from [JCenter](https://bintray.com/bintray/jcenter) and add configurations in the build.gradle file as follows.
 
-```
+```gradle
 allprojects {
     repositories {
         jcenter()
     }
 }
 
 dependencies {
-  compile 'com.linecorp:clayview:0.1.2'
+   implementation "com.linecorp:clayview:0.1.2"
```

Comment:
is there an extra space? 

```suggestion
  implementation "com.linecorp:clayview:0.1.2"
```

Reply:
Thank you for your review!
I removed the space.